roles/ocp_netobserv/tasks/verify.yml

```diff
@@ -0,0 +1,42 @@
+---
+- name: Confirm that pods are Running
+  community.kubernetes.k8s_info:
+    api_version: v1
+    kind: Pod
+    namespace: "netobserv"
+  register: pod_list
+  until: pod_list|json_query('resources[*].status.phase')|unique == ["Running"]
+  retries: 9
+  delay: 10
+  no_log: true
+
+- name: Confirm that PVCs are bound
+  community.kubernetes.k8s_info:
+    api_version: v1
+    kind: PersistentVolumeClaim
+    namespace: "netobserv"
+    name: "{{ item.name }}"
+  register: pvc
+  until:
+    - pvc is defined
+    - pvc.resources is defined
+    - pvc.resources | length > 0
```

Comment:
`pvc.resources | length > 0` -> `pvc.resources | length` to be consistent with the code above